Section_I/law4.tex

```diff
@@ -270,9 +270,10 @@
       legs and chest combined must be at least $0.06\cdot {H_{top}}^2$.
       The visible area of the one to five largest team markers on each side
       (left, right, front and back) must be at least $0.015\cdot {H_{top}}^2$.
-      If both teams cannot agree, which team colour to use, a coin will be
+      \removed{If both teams cannot agree, which team colour to use, a coin will be
       flipped an hour prior to the game to assign the team colours.
-      In the virtual competition, the color teams play in is randomly assigned and announced in the game plan.
+      In the virtual competition, the color teams play in is randomly assigned and announced in the game plan.}
```

Comment:
I don't think the part about the virtual competition should be removed as well - what would it be replaced by? There is no obvious left and right side of the field in the virtual competition, I believe.